libs/payments/ui/src/lib/client/components/PaymentSection.tsx

```diff
@@ -10,6 +10,9 @@ import { StripeWrapper } from './StripeWrapper';
 import { Localized } from '@fluent/react';
 
 interface PaymentFormProps {
+  config: {
+    stripePublicApiKey: string;
+  };
```

Comment:
Q: Out of curiosity, how often do we have to pass config information in the client components? Wondering if we could use a context provider so all child components will already have access to it?

Reply:
Oh yeah, that's a good idea! Let me put that in.